Makefile

```diff
@@ -66,7 +63,7 @@ endif
 
 .PHONY: build-all
 build-all:
-	go build ./...
+	go build -buildvcs=false ./...
```

Comment:
setting buildvcs to false, as this was resulting in [ci-go/ci-from-docker](https://github.com/Azure/ARO-RP/actions/runs/10617514593/job/29430527499?pr=3797) check failure with error `error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.`